private-eye.js

```diff
@@ -6,11 +6,36 @@
     styles.innerHTML = 'a.private-link::after { content: "\\1F512"; font-size: 0.75em; vertical-align: top; }';
     document.body.appendChild(styles);
 
+    var DEFAULT_MESSAGE = "This is a link to a private site, which may or may not be accessible to you.";
+    var hrefValue;
+    var linkMessage;
+
     opts.ignoreUrls.forEach(function(url) {
-      var anchors = document.querySelectorAll('a[href*="' + url + '"]');
+
+      hrefValue = url;
+      linkMessage = DEFAULT_MESSAGE;
```

Comment:
Minor gripe, but I would do these in an `else` below, just to make it more clear that only one of those sets of assignments will end up sticking.